backend/Pipfile

```diff
@@ -9,7 +9,7 @@ Flask-Migrate = "==2.1.1"
 Flask-Script = "==2.0.6"
 Flask-SQLAlchemy = "==2.3.2"
 gunicorn = "==19.7.1"
-"psycopg2" = "==2.7.5"
+"psycopg2-binary" = "==2.7.5"
```

Comment:
:rocket: nice, ill just verify with backend that it doesn't break anything so far and we'll pull it in

Reply:
Leaving this one to David, once that's good we can merge

Reply:
:lock: